models/DataObject/ClassDefinition/Data/CalculatedValue.php

```diff
@@ -232,7 +233,7 @@ public function getGetterCodeLocalizedfields(DataObject\Objectbrick\Definition|D
         $code .= '{' . "\n";
         $code .= "\t" . 'if (!$language) {' . "\n";
         $code .= "\t\t" . 'try {' . "\n";
-        $code .= "\t\t\t" . '$locale = \Pimcore::getContainer()->get("pimcore.locale")->getLocale();'  . "\n";
+        $code .= "\t\t\t" . '$locale = \Pimcore::getContainer()->get("' . LocaleServiceInterface::class . '")->getLocale();'  . "\n";
```

Comment:
we need a migration to re-generate the classes with calculated value fields otherwise it will generate deprecation warnings.

Reply:
@dvesh3 Done https://github.com/pimcore/pimcore/pull/16268/commits/418a3eed966a39ff1b098727c13b43b2dff141d8